unblob/handlers/archive/rar.py

```diff
@@ -0,0 +1,51 @@
+"""
+RAR Version 4.x
+https://codedread.github.io/bitjs/docs/unrar.html
+
+RAR Version 5.x
+https://www.rarlab.com/technote.htm#rarsign
+"""
+
+import io
+from typing import List, Optional
+
+import rarfile
+from structlog import get_logger
+
+from ...models import Handler, ValidChunk
+
+logger = get_logger()
+
+
+class RarHandler(Handler):
+    NAME = "rar"
+
+    YARA_RULE = r"""
+        strings:
+            $rar_magic_v4 = { 52 61 72 21 1A 07 00 }
+            $rar_magic_v5 = { 52 61 72 21 1A 07 01 00 }
+
+        condition:
+            $rar_magic_v4 or $rar_magic_v5
```

Comment:
We could simplify this YARA rule this way:

```suggestion
        strings:
            // RAR v4.x ends with 00, RAR v5.x ends with 01 00
            $magic_rar = { 52 61 72 21 1A 07 ( 00 | 01 00 ) }
        condition:
            $magic_rar
```

Reply:
Done in a separate PR: https://github.com/IoT-Inspector/unblob/pull/60